scripts/dashboard/RestoreWallet.vue

```diff
@@ -32,7 +32,7 @@ async function submit() {
     if (wif) {
         emit('import', wif, extsk);
     } else {
-        createAlert('warning', ALERTS.FAILED_TO_IMPORT);
+        createAlert('warning', ALERTS.INVALID_PASSWORD, 2000);
```

Comment:
The real problem is that  createAlert has a default `timeout = 0`:

`createAlert(level, message, timeout = 0) `

in file `alert.js`.

I suggest revert this line and setting a default value of 2000ms

Reply:
Good note, committing.